site/src/components/Mainabout.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { AboutStyleWrapper } from '../sections/mainabout.style'
+
+export const Mainabout = () => {
+  return (
+    <AboutStyleWrapper>
+        <div className='about'>
+            <div className="heading">About the <span className='green'>Service Mesh Patterns</span> Book</div>
```

Comment:
It's heading so use `h1` for it!
And, within it use the `span` for applying CSS to a particular text

Reply:
ah yes should hv used h1 , span is already been used :100: